src/test/java/edu/practice/spring_hw2/UserControllerTest.java

```diff
@@ -0,0 +1,56 @@
+package edu.practice.spring_hw2;
+
+import edu.practice.spring_hw2.entities.UserDTO;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class UserControllerTest {
+
+
+    @Test
+    public void testGetNewUser() {
```

Comment:
I will recommend you to read about `GIVEN WHEN THEN` style of representing tests